mds_extensions/README.md

```diff
@@ -158,8 +165,86 @@ The `/aggregate/trips` API will allow querying aggregate trips with the followin
+| `max_end_time` | [Timestamp](https://en.wikipedia.org/wiki/Unix_time) | filter for `trips` before the given time. |
+
+If the timestamps are not hour-bounded, the endpoint will round down to the most recent hour. If not provided, the
+endpoint will return the most recent hour for which it has data.
+
+[Top](#Table-of-Contents)
+
+## Aggregate Status Counts
+
+- The aggregate status counts endpoint will return a list of MDS status change counts by event type 
+  based on last reported status changes for vehicles in the specified sliding window.
+- The `data` property will provide information about `vehicle_type`.
+  - A `vehicle_type` list , will contain the hour_utc and event_type(s) count, which is the number of vehicles 
+    in that state.
+- The aggregations will be in one hour segments.
+
+**Endpoint:** `/aggregate/status_counts`
+
+**HTTP Method:** `GET`
+
+**Data payload format:** `{ "data": { "scooter": [] } }`, an array of objects with the following structure:
+
+| Field  | Type | Comments  |
+| -----  | ---- | --------  |
+| `data`  | Objects | Holds the status counts for each `vehicle type`.  |
+|  `lookback_days` |  Integer | Number of days to look back for vehicle status. |
+
+| Field  | Type | Comments  |
+| -----  | ---- | --------  |
+| `hour_utc`  | datetime [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) | Specifies the datetime when status count aggregation was performed.  |
+| `lookback_days` |  Integer | Number of days to look back for vehicle status, `default:` **2d** |
+
+**Example usage:**
+```
+# Request
+
+curl \
+-H "Accept: application/vnd.mds.provider+json;version=0.3" \
+-H "Authorization: Bearer $LIME_TOKEN" \
+-X GET \
+"https://data.lime.bike/api/partners/v1/mds/{city}/aggregate/status_counts?min_end_time=1602176400000&max_end_time=1602183600000&lookback_days=2"
+
+# Response
+
+{
+    "data": {
+        "scooter": [
+            {
+                "hour_utc": "2020-10-08T17:00:00.000Z",
+                "available": 156,
+                "reserved": 31,
+                "unavailable": 37,
+                "removed": 218
+            },
+            {
+                "hour_utc": "2020-10-08T18:00:00.000Z",
+                "available": 150,
+                "removed": 218,
+                "unavailable": 36,
+                "reserved": 34
+            }
+        ]
+    },
+    "lookback_days": 2
+}
+
+```
+
+[Top](#Table-of-Contents)
+
+### Status Counts - Query Parameters
+
+The `/aggregate/status_counts` API will allow querying aggregate status_counts with the following query parameters:
+
+| Field  | Type | Comments  |
+| -----  | ---- | --------  |
+| `min_end_time` | [Timestamp](https://en.wikipedia.org/wiki/Unix_time) | filter for `status_counts` after the given time.  |
+| `max_end_time` | [Timestamp](https://en.wikipedia.org/wiki/Unix_time) | filter for `status_counts` before the given time. |
+| `lookback_days`| Integer | Number of days to look back for vehicle status. |
```

Comment:
lookback_days could use more explanation. I would mention something like: Vehicles that have not had an event in this number of days will not appear in the counts.

Reply:
good point.  I will add a note